sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx

```diff
@@ -14,6 +14,8 @@ const DetailApplicationAddress = () => {
       return t("application.contact.mailingAddress")
   }
 
+  if (!listing.leasingAgentAddress) return <></>
```

Comment:
This is a partners page - is this an unrelated change?

Reply:
Yeah, sorry this is unrelated, it came up when I was testing and was throwing errors so I just fixed it. Same below.